client/src/components/Carousel.js

```diff
@@ -1,8 +1,38 @@
-import React from "react"
 
-const Carousel = ()=>{
-    return(
+import React, { useState, useEffect } from "react";
+import "../css/Carousel.css";
 
 
-    )
-}
\ No newline at end of file
+import imageOne from "../images/image 1.jpg";
+import imageTwo from "../images/image 2.jpg";
+import imageThree from "../images/image 3.jpg";
+
+
+const images = [imageOne, imageTwo, imageThree];
+
+const Carousel = () => {
+  const [curr, setCurr] = useState(0);
+  // current === images.length - 1 ? 0 : current + 1;
+
+  useEffect(() => {
+    setInterval(() => {
+      setCurr((current) => {
+        if (current === images.length - 1) {
+          return 0;
+        } else {
+          return current + 1;
+        }
+      });
+    }, 5000);
+  }, []);
+
+  return (
+    <div >
+      {images.map(
+        (image, index) => index === curr && <img src={image} alt="test" />
```

Comment:
I would suggest giving the image a meaningful name and use this name as alt text as well.